libleptonica/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>libleptonica</name>
+  <version>0.1.0</version>
+  <description>
+    ROS wrapper for leptonica (http://www.leptonica.org/).
+  </description>
+  <author email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</author>
+  <maintainer email="richard.bormann@ipa.fraunhofer.de">Richard Bormann</maintainer>
+  <license>LGPL, Leptonica has its own license</license>
```

Comment:
Leptonica is actually [BSD 2-clause](http://leptonica.org/about-the-license.html). Maybe this would be more descriptive!